src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -56,8 +59,13 @@ public CommandResult execute(Model model, CommandHistory history) throws Command
             set.add(ModelTypes.SCHEDULES_LIST);
         }
 
```

Comment:
model.updateFilteredExpensesList(PREDICATE_SHOW_ALL_EXPENSES); 
should be written before your method